client/src/components/game/gui/panels/chat/ChatSelectScope.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ChatState } from "../../../../../shared/state/States";
+import "./ChatSelectScope.scss";
+
+function ChatSelectScope({ updatePlaceHolder, setSendChatScope, closeSelectScopeDropdown }) {
+    const setChatScope = (scope) => {
+        ChatState.saveChatScope(scope);
+        setSendChatScope(scope);
+        closeSelectScopeDropdown();
+        updatePlaceHolder();
+    };
+
+    return (
+        <div className="chat-select-scope">
+            <p
+              className="scope-text local"
+              onClick={() => setChatScope(ChatState.CHAT_SCOPES.LOCAL.value)}
+            >
+                Local
```

Comment:
Need to use the selected language for these texts. `{ Utils.getTextDef("Local") }`.
I'll add the definitions to the translation spreadsheet.

Reply:
These translations have been added:

![image](https://user-images.githubusercontent.com/9785268/114321292-06c19a80-9b12-11eb-80ff-2eb19112dc35.png)